lib/Styled/Box.tsx

```diff
@@ -42,7 +42,7 @@ interface IBoxPropsBase {
   alignItemsFlexEnd?: boolean;
   rightSelf?: boolean;
   column?: boolean;
-  wrap?: boolean;
+  flexWrap?: boolean;
```

Comment:
Unreleated to this ticket. `wrap` is non-boolean standard HTML attribute so React throws an error in the console complaining about passing a boolean value for a non-boolean property. This fixes it.